x-pack/filebeat/input/httpjson/value_tpl_test.go

```diff
@@ -574,6 +574,17 @@ func TestValueTpl(t *testing.T) {
 			paramTr:     transformable{},
 			expectedVal: "my value",
 		},
+		{
+			name:  "func toJSON",
+			value: "[[ toJSON .last_response.body.paginationParams ]]",
```

Comment:
This test passes without the `toJSON` call since the parameter is a `mapstr.M` whose `String` method renders to JSON.

Reply:
Good point. I changed the test such that it will fail without `toJSON`. The code uses mapstr.M.Clone to convert inner objects to mapstr.M, but it does not look inside of `[]interface{}` for maps. So without toJSON you get

```
        	Error Trace:	value_tpl_test.go:609
        	Error:      	Not equal: 
        	            	expected: "[{\"id\":1234}]"
        	            	actual  : "[map[id:1234]]"
```